lib/trailblazer/developer.rb

```diff
@@ -1,8 +1,11 @@
 require "trailblazer/developer/version"
+require "dry-configurable"
+require "logger"
 
 module Trailblazer
   module Developer
-    # Your code goes here...
+    extend Dry::Configurable
+    setting(:logger, Logger.new($stdout), reader: true )
```

Comment:
Not a fan of a global var, that `logger` could sit in `flow_options`!!!

Reply:
It sitting inside `Trailblazer::Developer`